R/html_dependencies.R

```diff
@@ -526,3 +526,16 @@ html_dependency_katex <- function(href = NULL) {
     )
   )
 }
+
+#' @rdname html-dependencies
+#' @export
```

Comment:
I guess this is fine because `devtools::document()` has added a proper item to the NAMESPACE.
https://github.com/rstudio/rmarkdown/pull/2313/files#diff-b6b6854a02ae177f8860f49654ff250c1554d021ae434b6efdbe99d00bdc6055R30

Reply:
Yes running `devtools::document()` is what I meant. Thanks !